integration/tests/verification/streamNegotiation_test.go

```diff
@@ -0,0 +1,26 @@
+package verification
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+// TestVerificationStreamNegotiationSuite enables gzip stream compression only between execution and verification nodes, while the
+// rest of network runs on plain libp2p streams. It evaluates that network operates on its happy path concerning verification functionality.
+func TestVerificationStreamNegotiationSuite(t *testing.T) {
+	s := new(VerificationStreamNegotiationSuite)
+	s.preferredUnicasts = "gzip-compression" // enables gzip stream compression between execution and verification node
```

Comment:
shouldn't `gzip-compression` be a constant defined in the main code?

Reply:
Addressed in: https://github.com/onflow/flow-go/pull/1812/commits/60acaadd40153ddcb231befaa72e25481389e140